jdk-tool-plugin/src/main/java/hudson/tools/JDKInstaller.java

```diff
@@ -200,7 +208,7 @@ public void install(Launcher launcher, Platform p, FileSystem fs, TaskListener l
             }
 
             int exit = starter
-                    .stdin(new ByteArrayInputStream("yes".getBytes())).stdout(out)
+                    .stdin(new ByteArrayInputStream("yes".getBytes(Charset.defaultCharset()))).stdout(out)
```

Comment:
`StandardCharsets.ASCII`?

Reply:
Yeah, not sure what I was thinking here.